ql/src/test/queries/clientpositive/prepare_plan_func_expr.q

```diff
@@ -0,0 +1,12 @@
+--! qt:dataset:src
+--! qt:dataset:alltypesorc
+--SORT_QUERY_RESULTS
+
+explain prepare pint from select sum(ctinyint)/count(ctinyint) as ag from alltypesorc where cint <= ?  and cbigint <= ? and cfloat != ? group by cint;
+
+prepare pint from select sum(ctinyint)/count(ctinyint) as ag from alltypesorc where cint <= ?  and cbigint <= ? and cfloat != ? group by cint;
+
+explain execute pint using 100, 5000000, 0.023;
+
+execute pint using 100, 5000000, 0.023;
+
```

Comment:
Small nits:
1. Maybe add `explain cbo` plans too.
2. Since we want to just test if execute statements run, we don't need a lot of output rows anyway. So, maybe reduce the number of output rows by only using equality or IN operators. Also include those columns (cint, cbigint, cfloat) in the output.
3. If you want to test !=, <=, >= operators, add more tests with counts.

These are not required btw, and the change looks good to me.

Reply:
Thanks for the review @soumyakanti3578 , Can you please look at the changes in the test file and verify?

Reply:
Thanks, LGTM!